scrapy/extensions/corestats.py

```diff
@@ -24,7 +24,11 @@ def spider_opened(self, spider):
         self.stats.set_value('start_time', datetime.datetime.utcnow(), spider=spider)
 
     def spider_closed(self, spider, reason):
-        self.stats.set_value('finish_time', datetime.datetime.utcnow(), spider=spider)
+        finish_time = datetime.datetime.utcnow()
+        elapsed_time = finish_time - self.stats.get_value('start_time')
+        elapsed_time_seconds = elapsed_time.seconds + elapsed_time.microseconds/1000000
```

Comment:
In Python 2 `elapsed_time.microseconds/1000000` is always 0

Reply:
I think this is failing for python 2.7, what should I modify then @kmike ?